admin/client/src/styles/legacy/_style.scss

```diff
@@ -1614,21 +1614,28 @@ form.member-profile-form {
 }
 
 .cms .cms-panel-padded {
-		&.ReportAdmin {
-      .ss-gridfield-buttonrow {
-        margin-left: -20px;
-        margin-right: -20px;
-      }
+  &.ReportAdmin {
+    padding: $grid-gutter-width-half;
+    width: 100% !important;
 
-      > fieldset {
-  				padding: 0 $grid-gutter-width-half;
-  		}
+    > .message {
+      margin-bottom: $grid-gutter-width-half;
     }
 
-		.cms-content-view {
-			padding: 0 $grid-gutter-width-half $grid-y*1.5;
-		}
+    > fieldset {
+      margin-top: -$grid-gutter-width-half;
+    }
+
+    .ss-gridfield-buttonrow {
+      margin-left: -20px;
+      margin-right: -20px;
```

Comment:
Instead of all of this negative margin malarkey, we should simply remove `.cms-panel-padded` from the container here, right?